jwst/tests_nightly/general/miri/test_bias_drift2.py

```diff
@@ -0,0 +1,41 @@
+import os
+import pytest
+from astropy.io import fits as pf
+from jwst.refpix.refpix_step import RefPixStep
+
+from ..helpers import add_suffix
+
+pytestmark = [
+    pytest.mark.usefixtures('_jail'),
+    pytest.mark.skipif(not pytest.config.getoption('bigdata'),
+                       reason='requires --bigdata')
+]
+
+def test_refpix_miri2(_bigdata):
+    """
+
+    Regression test of refpix step performed on MIRI data.
+
+    """
+    output_file_base, output_file = add_suffix('refpix2_output.fits', 'refpix')
+
+    try:
+        os.remove(output_file)
```

Comment:
I would argue that this is no longer necessary given that `pytest` runs each test in its own `tmpdir` sub-directory.